### ISSUE TITLE
Linux idea for article "Operate a Dogecoin Node"

### ISSUE THREAD

**Issue by alamshafil** [dogecoinfoundation/dogecoin.com#126]
When installing Dogecoin Core on a Linux machine using the offical tarballs, it may be useful if the binaries (dogecoin-qt, dogecoind, etc) are installed to the system `PATH` in `/usr/local/bin`, this means said binaries can be called from any path in the terminal and makes it easier to setup init system (systemd, etc) to manage dogecoind.

```bash
sudo install -m 0755 -o root -g root -t /usr/local/bin dogecoin-1.14.6/bin/*
``` 

We can also provide information on how to keep dogecoind running after a restart (systemd, cron, etc).

**Comment by MishaBoar** [comment]
I agree, we need to expand and revise the Linux section, this was always in the cards and now it is more important than ever. The Linux installation instructions are inherited from the open source Bitcoin Core installation instructions.
